app/routes/events/view/sessions/create.js

```diff
@@ -23,5 +23,10 @@ export default Route.extend({
       tracks       : await eventDetails.query('tracks', {}),
       sessionTypes : await eventDetails.query('sessionTypes', {})
     };
+  },
+  resetController() {
```

Comment:
The first argument of this is `controller`. Use that instead of `this.controller`